src/background.ts

```diff
@@ -73,7 +73,9 @@ async function generateQuery(message: { context: string, url: string, dims: [num
       reader.onerror = () => reject(reader.error);
       reader.readAsDataURL(blob);
     });
-  }).then(image => {
+  }).then(async image => {
+    const debugMode = await isDebugModeEnabled();
+    const capabilities = debugMode ? [CAPABILITIES.debugMode] : [];
```

Comment:
This will likely need to be replaced with `push` as we get more capabilities to add. Optional, but maybe:
```suggestion
    const capabilities = [];
    if (debugMode) {
        capabilities.push(CAPABILITIES.debugMode);
    }
```